### PR TITLE
Cudf build fix [skip ci]

### PR DESCRIPTION
**Pull request by razajafri**
This fixes the broken build as a result of an incomplete check-in.

### CODE REVIEW

**Review by nartal1 — APPROVED** [review]

**Comment by nartal1** [comment]
LGTM.

**Comment by razajafri** [comment]
build

File: java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java [hunk 1]
```diff
@@ -4005,7 +4005,7 @@ void testReplaceLeafNodeInList() {
 
   @Test
   void testReplaceLeafNodeInListWithIllegal() {
-    assertThrows(IllegalArgumentException.class, () -> {
+    assertThrows(AssertionError.class, () -> {
```
Comment (gerashegalov) [inline java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java:4008]: can we also assert on the content of the expected exception message similar to https://stackoverflow.com/questions/40268446/junit-5-how-to-assert-an-exception-is-thrown.

**Comment by razajafri** [comment]
rerun tests

File: java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java (hunk 1 above)
Comment (razajafri) [reply java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java:4008]: I have updated. Can you see the change?

Comment (gerashegalov) [reply java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java:4008]: you may not have pushed yet @razajafri

**Comment by razajafri** [comment]
closed in favor of #7542

**Pull request closed without merging**